code/game/objects/items/cards_ids.dm

```diff
@@ -200,9 +200,10 @@
 	icon_state = "pmc"
 	registered_name = "The Corporation"
 	assignment = "Corporate Mercenary"
-	New()
-		access = get_all_centcom_access()
-		..()
+
+/obj/item/card/id/pmc/New()
```

Comment:
all of these cards could be init - not sure if you want to refactor it now or later